src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.ts

```
@@ -9,4 +9,16 @@ import { SidebarCardElement, SidebarTypes } from 'app/types/sidebar';
 export class SidebarCardItemComponent {
     @Input() sidebarItem: SidebarCardElement;
     @Input() sidebarType?: SidebarTypes;
+    @Input() groupKey?: string;
+
+    removeChannelPrefix(name: string): string {
+        const prefixes = ['exercise-', 'lecture-', 'exam-'];
+        const channelTypes = ['exerciseChannels', 'lectureChannels', 'examChannels'];
+        for (const prefix of prefixes) {
+            if (name && name.startsWith(prefix) && channelTypes.includes(<string>this.groupKey)) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)